pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4", "cython<3", "numpy"]
```

Comment:
Changes to attribute names mean that Pywr can't now be built with cython<3 so this should probably be updated to "cython>=3"

Reply:
Done